buildscript/src/main/kotlin/Dependencies.kt

```diff
@@ -11,6 +11,7 @@ object Dependencies {
         val espresso = "3.2.0"
         val mockito = "3.3.3"
         val detekt = "1.10.0"
+        val coroutines = "1.3.7"
```

Comment:
I feel that version upgrade should be in separate commit (PR better, because of squash);
at least mention it in resulting commit message (+PR title)